jenkins-dcp.planx-pla.net/portal/gitops.json

```diff
@@ -300,6 +300,46 @@
       },
       "nodeCountField": "case_id"
     },
```

Comment:
can you remove the `arrangerConfig` before this line since it is not needed anymore ?

Reply:
On that topic:
@mfshao @haraprasadj , should we also remove it from:
https://github.com/uc-cdis/data-portal/blob/master/data/config/default.json#L191
??

Reply:
Thanks for pointing it out, I missed that. Will take up the along with the cleanup of other envs portal configs